main/src/main/java/cgeo/geocaching/utils/ActionBarUtils.java

```diff
@@ -74,26 +72,15 @@ private static void showActionBarSpacer(@NonNull final Activity activity, final
         activity.findViewById(R.id.actionBarSpacer).setVisibility(showSpacer ? View.VISIBLE : View.GONE);
     }
 
-    private static void setStableLayout(@NonNull final AbstractNavigationBarActivity activity, final boolean showSpacer) {
-        if (showSpacer) {
-            activity.getWindow().getDecorView().setSystemUiVisibility(View.SYSTEM_UI_FLAG_LAYOUT_STABLE);
```

Comment:
I maybe found a different solution: don't use `setSystemUiVisibility(View.SYSTEM_UI_FLAG_LAYOUT_STABLE);`

It seems that with the insets, this maybe now obsolete. I tested it with Android 9 / API 28, but I am not sure, if this is the correct approach.

@moving-bits / @eddiemuc do you have any advice?